client-ts/package.json

```diff
@@ -18,7 +18,8 @@
     "rollup-plugin-commonjs": "^8.2.6",
     "rollup-plugin-node-resolve": "^3.0.2",
     "rollup-plugin-sourcemaps": "^0.4.2",
-    "typescript": "^2.6.2",
+    "tslint": "^5.9.1",
```

Comment:
@Eilon We're adding a new dev dependency
@anurse will submit an OSS report for it once approved

Reply:
Sounds good!